app/helpers/middleware_messaging_helper/textual_summary.rb

```diff
@@ -0,0 +1,23 @@
+module MiddlewareMessagingHelper::TextualSummary
+  include TextualMixins::TextualName
+  #
+  # Groups
+  #
+
+  def textual_group_properties
+    %i(name nativeid)
```

Comment:
Good question, I tried fetching type from hawkular inventory resource, but I am unable to access `type` value in `refresh_parser.rb`. I looked in [entities.rb](https://github.com/hawkular/hawkular-client-ruby/blob/56c16c41f1074cc7599e5c3be81f76f1291d13c2/lib/hawkular/inventory/entities.rb#L61) and it looks like there's missing `attr_reader :type`, hence no public access to this property.

However I can do `to_h` and read the type variable. So what should I do? New PR in hawkular-ruby-client` and make type variable publicly accessible (make it part of the API) or just use `to_h` and access type this way?